src/docks/subtitlesdock.cpp

```diff
@@ -360,6 +415,12 @@ void SubtitlesDock::setupActions()
     connect(action, &QAction::triggered, this, &SubtitlesDock::generateTextOnTimeline);
     Actions.add("subtitleGenerateTextAction", action, windowTitle());
 
+    action = new QAction(tr("Transcribe Audio..."), this);
```

Comment:
What do you think about calling it "Speech-to-Text..." or "Create from Audio..." as that might be more obvious.  "Transcribe" seems rather high-brow.

Reply:
I agree. And "Speech-to-Text" also maps to the label in the dialog to "Select tracks with speech".

Reply:
Done